frontends/infinite-corridor/src/components/FieldAvatar.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { render, screen } from "@testing-library/react"
+
+import { makeField } from "../api/fields/factories"
+import FieldAvatar from "./FieldAvatar"
+
+describe("Avatar", () => {
+  it("Displays a small avatar image for the field", async () => {
+    const field = makeField()
+    render(<FieldAvatar field={field} imageSize="small" />)
+    const img = screen.getByAltText(`Channel avatar for ${field.title}`)
```

Comment:
**Suggestion:** I think we should get rid of the alt text, and replace this with `getByRole("img")`.

I am by no means an accessibility expert, but my understanding is that alt text is better left blank if it does not add meaningful context. Like on the field page, a screenreader would currently read, I believe

> "Channel avatar for physics. Physics".

Better (I think?) just to say "Physics".

Related: https://silktide.com/blog/is-alt-text-making-your-site-less-accessible/

_Actually: Probably best to make this component accept `alt` as a prop. Then if we ever use the avatar as an unlabeled link, we can use pass an alt tag for it. But... we don't need that right now._